sockets/1.2/modules/ROOT/pages/sockets-documentation.adoc

```diff
@@ -542,14 +548,20 @@ Listens for socket connections of the given protocol in the configured host and
 [[StreamingProtocol]]
 === Streaming Protocol
 
+This protocol allows the socket’s Send operation to return a message with the original InputStream as payload.
+
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
 | Rethrow Exception On Read a| Boolean | Indicates if the exception should be rethrown if there is a failure while reading | false |
 |===
 
 [[XmlMessageProtocol]]
-=== Xml Message Protocol
+=== XML Message Protocol
+
+This protocol reads streaming XML documents. The only requirement is that each document includes an XML declaration at the beginning of the document in the form of <?xml…​..
+Data is read until a new document is found or until there is no more currently available data. For slower networks, XmlMessageEofProtocol might be more reliable.
+Also, because the default character encoding for the platform is used to decode the message bytes when looking for the XML declaration, some caution with message character encodings is warranted.
```

Comment:
```suggestion
Also, because the default character encoding for Anypoint Platform is used to decode the message bytes when looking for the XML declaration, some caution with message character encodings is warranted.
```